src/subcommand.ts

```diff
@@ -12,12 +12,12 @@ export class Subcommand {
   /**
    * This subcommand's description.
    */
-  public description: string = "";
+  public description = "";
```

Comment:
why are the types removed?

Reply:
Linter pulled this up, throws errors for properties that explicitly have the type even when it’s inferred

Reply:
yea the linter disagrees with strongly typed, inferred variables like this